website/data/docs-nav-data.json

```
@@ -767,7 +767,20 @@
       },
       {
         "title": "Configuration",
-        "path": "agent/options"
+        "routes": [
+          {
+            "title": "General",
+            "path": "agent/config"
+          },
+          {
+            "title": "CLI Reference",
+            "path": "agent/config/agent-config-cli"
+          },
+          {
+            "title": "Configuration Reference",
+            "path": "agent/config/agent-config-files"
```

What do you think about shortening these paths to this?

```suggestion
            "path": "agent/config/cli-flags"
          },
          {
            "title": "Configuration Reference",
            "path": "agent/config/config-files"
```

We already have agent and config in the path, so we might not need it in the filename as well.

---

Makes a lot of sense - updated!